src/components/popup.js

```diff
@@ -169,7 +170,7 @@ function Popup(props) {
   );
 
   // If eventManager does not exist (using with static map), listen to React event
-  const onReactClick = context.eventManager ? null : onClick;
+  const onReactClick = useCallback(e => !context.eventManager && onClick(e, thisRef), []);
```

Comment:
Sorry, one more thing...

```suggestion
  const onReactClick = useCallback(e => !context.eventManager && onClick(e, thisRef), [context.eventManager]);
```